.github/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 31. Enhancement - Adjustments to Gatwick (EGKK) SMR (stands, Z1, grass) - thanks to @hsugden (Harry Sugden)
 32. AIRAC (1905) - Updated Danger Area EG D406 - thanks to @hsugden (Harry Sugden)
 33. AIRAC (1905) - Updated Heathrow (EGLL) runway track and threshold coordinates - thanks to @hsugden (Harry Sugden)
+34. AIRAC (1905) - Updated Edinburgh SMR (EGPH) - thanks to @nathaninnes (Nathan Innes)
```

Comment:
This should go at the top of the file - under a heading

' # Changes from release 2019/05 to 2019/06" '